metastor/metastoreOperations/samples/object-registration/post_bo_data_notification_registration_template.xml

```diff
@@ -15,7 +15,7 @@
 -->
 <businessObjectDataNotificationRegistrationCreateRequest>
     <businessObjectDataNotificationRegistrationKey>
-        <namespace>METASTOR</namespace>
+        <namespace>{{MDL_METASTOR_NAMESPACE}}</namespace>
```

Comment:
does this mean we need to change herd-mdl metastor code accordingly?

Reply:
yes, Sid knows about it. We have to make these changes everytime we were these scripts.